Text/PrettyPrint/Leijen/Text.hs

```diff
@@ -126,7 +126,9 @@ import Data.String (IsString (..))
 import System.IO   (Handle, hPutChar, stdout)
 
 import           Data.Int               (Int64)
+import           Data.List              (intersperse)
 import           Data.Monoid            (Monoid (..), (<>))
+import qualified Data.Text              as Ts
```

Comment:
Can the import be `ST` or `TS` please? Having a lowercase there looks a bit odd.